ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java

```diff
@@ -417,8 +417,9 @@ protected int determineActiveToolState() {
     }
 
     private int determineLanguageState(final int languageIndex) {
-        if (mManifests.get(languageIndex) != null) {
-            return STATE_LOADED;
+        Manifest manifest = mManifests.get(languageIndex);
+        if (manifest != null) {
+            return Manifest.Type.TRACT == manifest.getType() ? STATE_LOADED : STATE_NOT_FOUND;
         } else if (isSyncToolsDone() && mTranslations.indexOfKey(languageIndex) >= 0 &&
```

Comment:
@frett so are you saying along with this Type check I should check to see if a category or AemImport exist?